Create ui-c8y-1024-5-2-truncate-datapoint-selector-in-legacy-data-graph-12464-graft-release-cd.md (#4943)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1024-5-2-truncate-datapoint-selector-in-legacy-data-graph-12464-graft-release-cd.md

---
date: ""

> title: Truncate datapoint selector in legacy data graph

Reviewer: ```suggestion
title: Fixed layout issue of datapoint selector in legacy Data graph widget
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66855
version: 1024.5.2
---

> Fixed long text overflowing the datapoint selector dropdown in the legacy data graph widget; it now truncates.

Reviewer: ```suggestion
Long text was overflowing the datapoint selector dropdown in the legacy "Data graph" widget. This issue has been fixed, and it now truncates.
```

---
date: ""

> title: truncate datapoint selector in legacy data graph (#12464) [GRAFT][release/cd] (#12572)

Reviewer 2: ```suggestion
title: Truncate datapoint selector in legacy data graph
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66855
version: 1024.5.2
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [fix(Web SDK): [MTM-66855] truncate datapoint selector in legacy data
graph
(#12464)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/12464)

<!--- Backport version: unknown -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-66855]:
https://cumulocity.atlassian.net/browse/MTM-66855?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

> Co-authored-by: Carlos Ceia <carlos.ceia@cumulocity.com>

Reviewer 2: ```suggestion
Fixed long text overflowing the datapoint selector dropdown in the legacy data graph widget; it now truncates.
```
